.github/workflows/test.yml

```diff
@@ -335,25 +336,63 @@ jobs:
         with:
           name: verdaccio-log-${{ matrix.template }}
           path: packages/pwa-kit-create-app/local-npm-repo/verdaccio-windows-${{ matrix.template }}.log
-  lighthouse:
+
+  # NOTE: Commenting out this lighthouse test because it's run against a _local_ server that skews the performance metrics to be better than they really are.
+  # TODO: perhaps run lighthouse in another workflow (e2e one) that deploys to MRT and test against that.
+  # lighthouse:
```

Comment:
I think as it is currently (and previously), the lighthouse test result is not useful because the performance score is too high (in the 90s). And it's because we were running the test against a local dev server.

Reply:
Since we have a long-running demo site deployed, should we run these test against a deployed site ?

Reply:
Yeah, I think it makes sense to run them in the E2E workflow.